N/Notcurses/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+using BinaryBuilder
+
+name = "Notcurses"
+version = v"3.0.0"
+sources = [
+    ArchiveSource("https://github.com/dankamongmen/notcurses/archive/refs/tags/v3.0.0.tar.gz",
+                  "b4839108ca8f9aef31d2f537485cbd878356bbcd4ad4f7a4dd19e72201d01cee"),
+    DirectorySource("./bundled"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/notcurses-*/
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/repent.patch
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/0001-compat-only-declare-strndup-on-windows-c-2428.patch
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/librt.patch
+
+
+mkdir build && cd build
+
+FLAGS=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
+       -DCMAKE_INSTALL_PREFIX=${prefix}
+       -DCMAKE_BUILD_TYPE=Release
+       -DBUILD_SHARED_LIBS=ON
+       -DUSE_MULTIMEDIA=none
+       -DUSE_DOCTEST=off
+       -DUSE_PANDOC=off
+       -DUSE_STATIC=off
+       -DUSE_QRCODEGEN=off
+       -DUSE_POC=off
+       )
+
+cmake .. "${FLAGS[@]}"
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line.
+platforms = supported_platforms()
+
+# The products that we will ensure are always built.
+products = [
+    LibraryProduct("libnotcurses", :libnotcurses)
+    LibraryProduct("libnotcurses-core", Symbol("libnotcurses-core"))
+
```

Comment:
Maybe use
```suggestion
    LibraryProduct("libnotcurses-core", :libnotcurses_core)
```
?  Otherwise it'd be a bit annoying to reference the variable with `var"libnotcurses-core"`